stdlib/platform.pyi

```diff
@@ -15,19 +16,32 @@ def system_alias(system: str, release: str, version: str) -> tuple[str, str, str
 def architecture(executable: str = sys.executable, bits: str = "", linkage: str = "") -> tuple[str, str]: ...
 
 if sys.version_info >= (3, 9):
+    # This class is not exposed. It calls itself platform.uname_result_base.
+    # At runtime it only has 5 fields.
     @type_check_only
     class _uname_result_base(NamedTuple):
         system: str
         node: str
         release: str
         version: str
         machine: str
+        # This base class doesn't have this field at runtime, but claiming it
+        # does is the least bad way to handle the situation. Nobody really
+        # sees this class anyway. See #13068
+        processor: str
 
+    # uname_result emulates a 6-field named tuple, but the processor field
+    # is lazily evaluated rather than being passed in to the constructor.
     class uname_result(_uname_result_base):
+        if sys.version_info >= (3, 10):
+            __match_args__ = ("system", "node", "release", "version", "machine")  # pyright: ignore[reportAssignmentType]
+
+        def __new__(_cls, system: str, node: str, release: str, version: str, machine: str) -> Self: ...
         @property
         def processor(self) -> str: ...
```

Comment:
Do we need the override here? The property exists on the base class in the stub, and the override in the subclass here has exactly the same type

Reply:
Huh. Weird, but sure!